accounts-cluster-bench/src/main.rs

```diff
@@ -227,6 +229,264 @@ fn make_close_message(
+                        stats.success += 1;
+                        stats.total_success_time_us += rpc_time.as_us();
+                    }
+                    Err(e) => {
+                        rpc_time.stop();
+                        stats.total_errors_time_us += rpc_time.as_us();
+                        stats.errors += 1;
+                        if last_error.elapsed().as_secs() > 2 {
+                            info!("get_slot error: {:?}", e);
+                            last_error = Instant::now();
+                        }
+                    }
+                }
+            }
+            RpcBench::GetMultipleAccounts => {
+                process_get_multiple_accounts(
+                    max_closed,
+                    max_created,
+                    &mut stats,
+                    &mut last_error,
+                    base_keypair_pubkey,
+                    program_id,
+                    client,
+                );
+            }
+            RpcBench::GetProgramAccounts => {
+                let mut rpc_time = Measure::start("rpc-get-program-accounts");
+                match client.get_program_accounts(program_id) {
+                    Ok(accounts) => {
+                        rpc_time.stop();
+                        stats.success += 1;
+                        stats.total_success_time_us += rpc_time.as_us();
+                        if thread_rng().gen_ratio(1, 100) {
+                            info!("accounts: {} first: {:?}", accounts.len(), accounts.first());
+                        }
+                    }
+                    Err(e) => {
+                        rpc_time.stop();
+                        stats.errors += 1;
+                        stats.total_errors_time_us += rpc_time.as_us();
+                        if last_error.elapsed().as_secs() > 2 {
+                            info!("get-program-accounts error: {:?}", e);
+                            last_error = Instant::now();
+                        }
+                    }
+                }
+            }
+            RpcBench::GetTokenAccountsByOwner => {
+                let filter = TokenAccountsFilter::Mint(*mint.as_ref().unwrap());
+                match client.get_token_accounts_by_owner(program_id, filter) {
+                    Ok(_accounts) => {
+                        stats.success += 1;
+                    }
+                    Err(e) => {
+                        stats.errors += 1;
+                        if last_error.elapsed().as_secs() > 2 {
+                            info!("get-token-accounts error: {:?}", e);
+                            last_error = Instant::now();
+                        }
+                    }
+                }
+            }
+            RpcBench::Version => match client.get_version() {
+                Ok(_r) => {
+                    stats.success += 1;
+                }
+                Err(_e) => {
+                    stats.errors += 1;
+                }
+            },
+        }
```

Comment:
Why not count `total_success_time_us` and `total_errors_time_us` for either of these variants?

Reply:
just an oversight, i'll add them